client/my-sites/theme/main.jsx

```diff
@@ -128,6 +141,10 @@ const ThemeSheet = React.createClass( {
 		const img = <img className="themes__sheet-img" src={ this.props.screenshot + '?=w680' } />;
 		return (
 			<div className="themes__sheet-screenshot">
+				<a className="themes__sheet-preview-link" onClick={ this.togglePreview } >
+					<Gridicon icon="external" size="18" />
```

Comment:
Yeah sorry, I keep missing that. Fixed in 049df2b3e34d8da944150b24b37b7bac7df51f21.